### ISSUE TITLE
Dtale has no toolbar to view longer values

### ISSUE THREAD

**Issue by Azrael1** [man-group/dtale#477]
I am trying to view a dataset, with large text values. Dtale increases the column width so much, that it is a pain to scroll to the right. 

Can there be an option to change the default column width?

Also, a toolbar that shows the current cell's value, would be quite useful(similar to excel). This would also eliminate the need for really big column sizes.

**Comment by aschonfeld** [comment]
Sorry I thought I had left a little tooltip with the full value, but I think that only happens if you use the string formatting functionality to trim the value and an ellipses exists.  Until I can build something more permanent (like what is in excel) I can update the code to have a tooltip display the full value a column is over a certain length (100 characters sound good?)

**Comment by Azrael1** [comment]
Sure, and thank you for the prompt response. 

Dtale is amazing btw!

**Comment by aschonfeld** [comment]
@Azrael1 here's what I have so far.  This is using a giant string of 1369 characters and I just created a dataframe with random substrings of this long string:
```
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet. Quisque rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. Nam quam nunc, blandit vel, luctus pulvinar, hendrerit id, lorem. Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,
```

https://user-images.githubusercontent.com/11547371/118531205-94953480-b713-11eb-8058-4793c1b3444d.mov

So the idea is that you can adjust the maximum width of the columns in your grid using that slider or the text input from the ribbon menu.  Once you updated the value the grid is re-drawn.  After its been re-drawn you can still drag the column widths past that maximum if you'd like.  You can also turn off the maximum width by toggling that checkbox from the ribbon menu.

Not sure how long the text is that you're viewing but as you can see the tooltip hits some height issues depending on your browser.

**Comment by aschonfeld** [comment]
Now I'm going to work on trying to add a view at the top (similar to excel) which will appear when you double-click a cell for editing.  And it will be hidden when nothing is being edited.  Currently you can press "Esc" to exit editing.

**Comment by aschonfeld** [comment]
@Azrael1 getting a little further...

https://user-images.githubusercontent.com/11547371/118738050-179db400-b814-11eb-9d66-30c24d8c200c.mov

**Comment by Azrael1** [comment]
Thanks a bunch, this really helps. Should I close the issue then?

**Comment by aschonfeld** [comment]
I’ll keep it open until I do a release in a few days. Just need to add some test coverage

**Comment by aschonfeld** [comment]
@Azrael1 this has been released to pypi in v1.47.0 (it will be in conda-forge shortly).  Please throw your ⭐  on the repo if you haven't done so already.  And let me know if you hit any issues 🙏

**Comment by liebkne** [comment]
@aschonfeld Hi, I can't get cell text previewed as shown in you video when max-width is applied. Do you have any idea of this? Thanks.

**Comment by aschonfeld** [comment]
@liebkne sorry I just merged a fix for this issue and will hopefully have a release out sometime in the next couple days

**Comment by aschonfeld** [comment]
@liebkne this has been fixed in v1.53.0 If you haven't already, please take a second to toss your ⭐  on the repo.  Thanks 🙏
